resources/run_adf.sh

```diff
@@ -280,6 +297,43 @@ else
   fi
 fi
 
+manage_compute_node() {
+  if [ $ARG = $START ]; then
+    check_compute_node $MONGO_DOWNLOAD_DIR
```

Comment:
Here, we're checking if the compute node is running but don't check the return or change behavior based on that.  An example of how check_mongohoused is doing it [here](https://github.com/mongodb/mongo-odbc-driver/blob/eed6cce38e8bbbbc277ac6d916ba55f0350e76e8/resources/run_adf.sh#L284).